### PR TITLE
azure-pipelines: Add fixes to allow automate release using azure pipelines.

### PR DESCRIPTION
**Pull request by jcvenegas**
This fixes are based on the work to create a release job in [azure pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/)

Preview: 
https://dev.azure.com/vmjcarlos/kata-containers/_release

Fixes: #443

    Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
\test

File: obs-packaging/distros [hunk 1]
```diff
@@ -13,5 +13,5 @@ SLE_12_SP3::SUSE:SLE-12-SP3:GA::standard
 openSUSE_Leap_42.3::openSUSE:Leap:42.3::standard
 openSUSE_Leap_15.0::openSUSE:Leap:15.0::standard
 openSUSE_Tumbleweed::openSUSE:Factory::snapshot
-xUbuntu_16.04::Ubuntu:16.04::universe
+xUbuntu_16.04::Ubuntu:16.04::universe,universe-update,update
```
Comment (jodh-intel) [inline obs-packaging/distros]: ooi, why is this change not required for 18.04?

Comment (jcvenegas) [reply obs-packaging/distros]: seems that the package that they provide in the base are enough to build kata today.

**Pull request merged**